coalib/settings/ConfigurationGathering.py

```diff
@@ -293,6 +301,23 @@ def load_configuration(arg_list,
 
         sections = merge_section_dicts(sections, cli_sections)
 
+    remove_default_section(sections)
+
+    remove_default_section(to_save_sections)
+
+    str_log_level = str(sections['cli'].get('log_level', '')).upper()
+    logging.getLogger().setLevel(LOG_LEVEL.str_dict.get(str_log_level,
+                                                        LOG_LEVEL.INFO))
+
+    return to_save_sections, sections, targets
```

Comment:
can you look for a way to have `load_configuration` return the old tuple return type, and create a new function to return the new improved return type.

Otherwise `load_configuration` is broken in all existing uses of it, and most coala tools use it, and expect a tuple returned.